it.es5.whoisavailable.yml

```diff
@@ -0,0 +1,50 @@
+app-id: it.es5.whoisavailable
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: whoisavailable
+
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --env=PATH=/app/jre/bin:/app/bin:/usr/bin
+  - --env=JAVA_HOME=/app/jre
+
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+
+  - name: whoisavailable
+    buildsystem: simple
+    build-options:
+      env:
+        PATH: /app/bin:/usr/bin:/usr/lib/sdk/openjdk11/bin
+        JAVA_HOME: /usr/lib/sdk/openjdk11/jvm/openjdk-11
+    build-commands:
+      - install -Dm755 whoisavailable.sh /app/bin/whoisavailable
+      - install -Dm644 -t /app/share/it.es5.whoisavailable whois-available.jar
+      - install -Dm644 -t /app/share/icons/hicolor/scalable/apps/ it.es5.whoisavailable.svg
+      - install -Dm644 -t /app/share/applications it.es5.whoisavailable.desktop
+    sources:
+      - type: file
```

Comment:
none of these URL are versioned. Which mean that when the files get updated on server the build will break. It's a bad practice overall.

Don't you use a version control system?